.travis.yml

```diff
@@ -5,11 +5,10 @@ notifications:
   email: false
 before_install:
   - gem install xcpretty --no-rdoc --no-ri --no-document --quiet
+  - brew install sourcekitten
 script:
   - set -o pipefail
   - cd apple/diffreport
```

Comment:
are we supposed to issue this dir change if you removed the line below?

Reply:
Yep - I'm not aware of a way to run `swift` project commands from directories other than the directory itself.